unfoldingWord/translationCore
# Added an 'n' to traslation

#### This pull request addresses:

Single letter change

#### How to test this pull request:
Visual Inspection
1 run tC
2 visually inspect footer on several pages and verify correct spelling of translation


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3141)
